cdc/sink/mysql/mysql_syncpoint_store.go

```diff
@@ -209,6 +210,19 @@ func (s *mysqlSyncPointStore) SinkSyncPoint(ctx context.Context,
 		log.Error("sync table: begin Tx fail", zap.Error(err))
 		return cerror.WrapError(cerror.ErrMySQLTxnError, err)
 	}
+
+	// we set `tidb_enable_external_ts_read` = off for each txn,
+	// otherwise TiCDC may not be able to write data to downstream TiDB when
+	// the downstream tidb_enable_external_ts_read = on.
+	if err = s.setTidbEnableExternalTsReadOff(ctx, tx); err != nil {
```

Comment:
maybe impact the dc mysql sink performance

Reply:
it will affect the performance of mysql sink more than sync_store